docs/aerodromes/classc/Williamtown.md

```diff
@@ -21,6 +21,56 @@ WLM ADC owns the airspace within the WLM MIL CTR A (`SFC`-`A050`). This may be a
   <figcaption>WLM ADC Airspace</figcaption>
 </figure>
 
+### SID Selection
+Aircraft planned via **KEXAR**, **LIMLO**, **MONDO**, **UBSON**, shall be assigned the **Procedural SID** that terminates at the appropriate waypoint.
+Aircraft who are not planned via those points or who are negative RNAV may be assigned a RADAR SID or a visual departure. 
+
+#### Coded Departures (Fast Jets)
+Visual Departures are commonly requested by high performance military jets and often processed in the form of a coded departure (e.g. CLASSIC1 departure), which can be found in the AD2 Sups Page for YWLM. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east and west of Williamtown, avoiding civilian traffic inbound to Sydney. 
+
+| Tracking Gate | Departure | Initial Constraint | Route |
+| ----- | ---------| --------------- | ------------------------ |
+| G1-G2 | PEPPER 1 | `A080` at ERVEV | ERVEV G1/G2 (Transition) 
+| G3-G7 | CLASSIC 1| `F125` at G3-G7 |
+| G9    | RADAR    | 'F125' at G9    |
+| UKDIP | CLASSIC 1| 'A090' at UKDIP |
+| OLVEP | CLASSIC 1| 'F130' at OLVEP |
+
+!!! note
+    ACFT Departing RWY 12 for G3-7 are required to avoid SAWR on the Classic 1.
+    ACFT Departing on Classic 1 will be issued a direction of turn during take off clearance, preferablly to the south in order not to impede on the SAWR boundries on the departure.
```

Comment:
Is there any guidance on how these aircraft should track to avoid the weapons range?

Reply:
Typically either climb quickly up to F130 or depart to the south.